package-info.java

```diff
@@ -0,0 +1,6 @@
+/*
+* This unnamed package contains files to facilitate running the application.
+* Files include configuration and build files, along with a README.md file,
+* a jar file that can be run from the command line
+* and a Run.java file to run the application.
+*/
```

Comment:
This comment does not show up in the javadocs.
One likely contributing cause is the incorrect comment-opening "`/*`" rather than the required javadoc comment opening "`/**`".

Also, this Java package doesn't contain the configuration and build files, etc.  (Those are outside of the Java program and simply reside in the same directory.)  The package, at present, only contains the Run class.  (See the content of the javadocs produced by the project.)

Reply:
All the javadoc and comments should show up now.